root/static/scripts/edit/externalLinks.js

```diff
@@ -612,6 +615,10 @@ function isShortened(url) {
   });
 }
 
+function isMusicBrainz(url) {
+  return /^https?:\/\/([^/]+\.)?musicbrainz.org/.test(url);
```

Comment:
not that it matters unless someone tries to link to musicbrainzzorg but nitpick
```suggestion
  return /^https?:\/\/([^/]+\.)?musicbrainz\.org/.test(url);
```